src/util/config/config.cpp

```diff
@@ -732,10 +728,6 @@ namespace dxvk {
     { R"(\\bionic_commando\.exe$)", {{
       { "d3d9.maxFrameRate",                "60" },
     }} },
-    /* Need For Speed 3 modern patch            */
-    { R"(\\nfs3\.exe$)", {{
-      { "d3d9.enableDialogMode",            "True" },
```

Comment:
Ditto

Reply:
It's d3d8

Reply:
Originall d3d6, but it has a native d3d8 rendered with the "modern patch". It can also end up into d3d9 via nglide, but it makes more sense to have it under d3d8.